notebooks/quantile_plot.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.    sns.despine()
</pre><p>This should be called before pml.savefig else it wont work during saving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>